sakai-evaluation-impl/src/ddl/conversion/oracle/eval-1.4_to_eval-11.sql

```diff
@@ -24,3 +24,5 @@ create table EVAL_HIERARCHY_RULE (
     OPT varchar(10) not null,
     primary key (ID)
 );
+
+INSERT INTO EVAL_EMAIL_TEMPLATE (LAST_MODIFIED, OWNER, TEMPLATE_TYPE, SUBJECT, MESSAGE, defaultType, EID) VALUES (sysdate,'admin','Available Evaluatee','The Evaluation: ${EvalTitle} for ${EvalGroupTitle} is available to be taken','All information submitted to the Evaluation System is confidential. Instructors cannot identify which submissions belong to which students. Students are required to login to the system for the sole purpose of providing students access to the appropriate evaluations for their associated courses. Instructors can only view general statistics as allowed by the university. Please send privacy concerns to ${HelpdeskEmail}. \n\nAn evaluation (${EvalTitle}) for: ${EvalGroupTitle} is ready to be filled out by students.It has an open date of ${EvalStartDate} and is due by ${EvalDueDate} at the latest.\n\nYou may access the evaluation at:\n${URLtoTakeEval} \nIf the above link is not working then please follow the Alternate Instructions at the bottom of the message. \nEnter the site using your username and password.\n------------------------------------------------------------\nShould you encounter any technical difficulty in filling out the evaluation, please send an email to ${HelpdeskEmail} clearly indicating the problem you encountered. For any other concerns please contact your department.\n\nAlternate Instructions: \n1) Go to ${URLtoSystem} \n2) Enter your username and password and click on \'Login\' button. \n3) Click on \'Evaluation System\' in the left navigation menu under My Workspace. \n4) Click on \'${EvalGroupTitle}\' link under \'${EvalTitle}\'.. \n','Available Evaluatee',NULL);
```

Comment:
The `INSERT` statement bombs on Oracle for two reasons:

1. The quote escaping. Instead of \', the escape sequence is '' (double single quote). Oracle....
2. There's no autonumber in Oracle, so you have to explicitly provide a value for the primary key (ID column). However there's also no corresponding Oracle sequence to generate a number for you on the DB side. All the old templates were saved through Hibernate, which did the primary key generation on the Java side. Not quite sure how we want to handle this... Looking at the code in PreloadDataImpl.preloadEmailTemplate(), it should load all the templates in Java, check which one's aren't in the database, and then save those that aren't into the database. But this isn't working perhaps?

I also noticed a few very minor grammar inconsistencies:

* I would drop the colon from the subject line (and the corresponding sentence in the body), but that's just my opinion
* "Students are required to login to the system..." --> this should be "log in", two words
* "Instructors can only view general statistics as allowed by the university" --> "university" here should probably be changed to "institution"
* "... is ready to be filled out by students.It has an open date of ...." --> need a space between sentences here
* "Should you encounter any technical difficulty... clearly indicating the problem"--> difficulty should maybe be pluralized, difficulties; problem -> problem(s)
* "Enter your username and password and click on 'Login' button" --> Login should be two words now, and it should probably say "the 'Log In' button" rather than just log in button
* "... in the left navigation menu under My Workspace" --> should be "Home" rather than "My Workspace" likely
* there's two periods at the end of the last sentence before the new-line character